banking_system/funds_pln.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include "i_funds.h"
+
+namespace bank {
+    class FundsPLN: public IFunds {
+
+    public:
+        explicit FundsPLN(double const _amountOfMoney): IFunds(_amountOfMoney) {}
```

Comment:
amountOfMoney - for function/method/constructor arguments
_amountOfMoney - for private fields